src/index.html

```diff
@@ -11,7 +11,232 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()"
+  >
+      <fieldset class="block-form">
+        <legend>Personal information</legend>
+        <label
+          class="field-form"
+          for="surname"
+        >
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            minlength="3"
+            maxlength="25"
+            required
+          >
+        </label>
+
+        <label
+          class="field-form"
+          for="name"
+        >
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            minlength="3"
+            maxlength="15"
+            required
+          >
+        </label>
+
+        <label
+          class="field-form"
+          for="age"
+        >
+          How old are You? 
+          <input
+            type="number"
+            id="age"
+            value="12"
+            name="age"
+            min="12"
+            max="100"
+            required
+          >
+        </label>
+
+        <label
+          class="field-form"
+          for="born"
+        >
+          Full date of birth:
+          <input
+            type="date"
+            name="born"
+            id="born"
+          >
+        </label>
+
+        <label
+          class="field-form"
+          for="term"
+        >
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset class="block-form">
+        <legend>Registration</legend>
+        <label
+          class="field-form"
+          for="email"
+        >
+          E-mail:
+          <input type="email"
+            placeholder="email@example.com"
+            name="e-mail"
+            id="email"
+          >
+        </label>
+
+        <label
+          class="field-form"
+          for="password"
+         >
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            maxlength="12"
+            pattern=".{8,}"
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset class="block-form">
+        <legend>An interesting fact about you!</legend>
+        <div class="field-form">
+          Do you love cats?
+          <label
+            for="catsYes"
+          >
+            <input
+              type="radio"
+              id="catsYes"
+              name="cats"
+              value="yes"
+            >
+              Yes
+          </label>
+          <label
+            for="catsNo"
+          >
```

Comment:
```suggestion
          <label for="catsNo">
```
Если атрибут всего один - можно записывать вот так